src/cls/rgw/cls_rgw_ops.cc

```diff
@@ -388,6 +388,82 @@ void rgw_cls_bucket_update_stats_op::dump(Formatter *f) const
   encode_json("stats", s, f);
 }
 
+void rgw_cls_obj_remove_op::dump(Formatter *f) const {
+  list<std::string> k;
+  for (auto& entry : keep_attr_prefixes) {
+    k.push_back(entry);
+  }
+  ::encode_json("keep_attr_prefix", k, f);
+}
+
+void rgw_cls_obj_remove_op::generate_test_instances(list<rgw_cls_obj_remove_op *>& o) {
+  rgw_cls_obj_remove_op *r = new rgw_cls_obj_remove_op;
+  r->keep_attr_prefixes.push_back("111");
+  r->keep_attr_prefixes.push_back("222");
+  o.push_back(r);
+
+  o.push_back(new rgw_cls_obj_remove_op);
+}
+
+void rgw_cls_obj_store_pg_ver_op::dump(Formatter *f) const {
+  ::encode_json("attr", attr, f);
+}
+
+void rgw_cls_obj_store_pg_ver_op::generate_test_instances(list<rgw_cls_obj_store_pg_ver_op *>& o){
+  rgw_cls_obj_store_pg_ver_op *r = new rgw_cls_obj_store_pg_ver_op;
+  r->attr = "test";
+  o.push_back(r);
+
+  o.push_back(new rgw_cls_obj_store_pg_ver_op);
+}
+
+void rgw_cls_obj_check_attrs_prefix::dump(Formatter *f) const {
+  ::encode_json("check_prefix", check_prefix, f);
+  ::encode_json("fail_if_exist", fail_if_exist, f);
+}
+
+void rgw_cls_obj_check_attrs_prefix::generate_test_instances(list<rgw_cls_obj_check_attrs_prefix *>& o){
+  rgw_cls_obj_check_attrs_prefix *r = new rgw_cls_obj_check_attrs_prefix;
+  r->check_prefix = "test_prefix";
+  r->fail_if_exist = true;
+  o.push_back(r);
+
+  o.push_back(new rgw_cls_obj_check_attrs_prefix);
+}
+
+void rgw_cls_obj_check_mtime::dump(Formatter *f) const {
+  ::encode_json("mtime", mtime, f);
+  ::encode_json("type", int(type), f);
+  ::encode_json("high_precision_time", high_precision_time, f);
+}
+
+void rgw_cls_obj_check_mtime::generate_test_instances(list<rgw_cls_obj_check_mtime *>& o) {
```

Comment:
Hey @cbodley , I have reviewed these changes, and all seems fine but I have difficulties in **void rgw_cls_obj_check_mtime::generate_test_instances(list<rgw_cls_obj_check_mtime *>& o)** function.
Please check once